llvm/test/CodeGen/AMDGPU/wqm.mir

```diff
@@ -1,4 +1,4 @@
-# RUN: llc -march=amdgcn -mcpu=fiji -verify-machineinstrs -run-pass si-wqm -o -  %s | FileCheck %s
+# RUN: llc -mtriple=amdgcn -mcpu=fiji -verify-machineinstrs -run-pass si-wqm -o -  %s | FileCheck %s
```

Comment:
This looks like an unrelated change. Can you please commit it separately ( it is obvious, no need for review)?